django/db/backends/oracle/base.py

```diff
@@ -200,10 +200,14 @@ def _connect_string(self):
                              settings_dict['PASSWORD'], dsn)
 
     def get_connection_params(self):
+        # Encoding must be specify in connection parameters to support unicode
```

Comment:
"Specify encoding to support unicode in DSN."?
Maybe it would be better to to change `kwargs` -> `conn_params` and the existing `conn_params` to something like `user_params`. The `kwargs` name seems off to me. Sorry for the back and forth over a simple patch!